<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SFTP - Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="group_server">
     <property name="title">
      <string>Server</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_hostInput">
        <property name="text">
         <string>Host:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="layout_host">
        <item>
         <widget class="QLineEdit" name="input_host"/>
        </item>
        <item>
         <widget class="QLabel" name="label_port">
          <property name="text">
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="input_port">
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>22</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_username">
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="input_username"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Authentication:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="combo_auth">
        <item>
         <property name="text">
          <string>Password</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Key</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_password">
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="input_password">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_keyfile">
        <property name="text">
         <string>Keyfile</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="input_keyfile"/>
        </item>
        <item>
         <widget class="QPushButton" name="button_browse">
          <property name="text">
           <string>Browse...</string>
          </property>
          <property name="icon">
           <iconset theme="document-open">
            <normaloff/>
           </iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_passphrase">
        <property name="text">
         <string>Passphrase</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="input_passphrase">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Optional</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_location">
     <property name="title">
      <string>Location</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_folder">
        <property name="text">
         <string>Folder:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="input_folder">
        <property name="placeholderText">
         <string>Example: /imgs/</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_url">
        <property name="text">
         <string>URL:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="input_url">
        <property name="placeholderText">
         <string>Example: http://example.com/imgs/</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkbox_url_extension">
        <property name="checked">
         <bool>True</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_url_extension">
        <property name="text">
         <string>Include file extension in URL</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_url_result">
        <property name="text">
         <string>URL result:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_url_example">
        <property name="text">
         <string>ex</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_name">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="input_name">
        <property name="text">
         <string extracomment="Hlap"/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_name_result">
        <property name="text">
         <string>Name result:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_name_example">
        <property name="text">
         <string>ex</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>input_host</tabstop>
  <tabstop>input_port</tabstop>
  <tabstop>input_username</tabstop>
  <tabstop>combo_auth</tabstop>
  <tabstop>input_password</tabstop>
  <tabstop>input_keyfile</tabstop>
  <tabstop>button_browse</tabstop>
  <tabstop>input_passphrase</tabstop>
  <tabstop>input_folder</tabstop>
  <tabstop>input_url</tabstop>
  <tabstop>input_name</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
